nixos/modules/services/networking/networkmanager.nix

```diff
@@ -12,15 +12,16 @@ let
   configFile = writeText "NetworkManager.conf" ''
     [main]
     plugins=keyfile
+    dhcp=${cfg.dhcp}
+    dns=${cfg.dns}
 
     [keyfile]
-    ${optionalString (config.networking.hostName != "")
-      ''hostname=${config.networking.hostName}''}
     ${optionalString (cfg.unmanaged != [])
       ''unmanaged-devices=${lib.concatStringsSep ";" cfg.unmanaged}''}
 
     [logging]
-    level=WARN
+    level=${cfg.logLevel}
+    audit=${if config.security.audit.enable then "true" else "false"}
```

Comment:
Looks like this patch has already been integrated into master. Except for this line.

Reply:
Nice catch. There the audit bit as well as dbus config for dnsmasq.

Reply:
Yup